src/components/SimpleInput.js

```diff
@@ -0,0 +1,55 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const Input = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    type: React.PropTypes.string,
+    valid: React.PropTypes.bool
+  },
+
+  getDefaultProps () {
+    return {
+      type: 'text',
+      valid: true
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <input
+        {...this.props}
+        name={this.props.name}
+        style={Object.assign({}, styles.wrapper, this.props.style)}
+        type={this.props.type}
```

Comment:
:cookie: Very cool to know. I hadn't thought of that. Much more concise. I still think they should be included in propTypes, just to have a reference to what is expected/passed.